tests/knr1050.py

```diff
@@ -86,7 +86,7 @@ def test_GIVEN_set_concentration_via_ioc_WHEN_ramp_command_sent_via_ioc_THEN_cor
         self.ca.set_pv_value("COMP:B:SP", expected_concentrations[1])
         self.ca.set_pv_value("COMP:C:SP", expected_concentrations[2])
         self.ca.set_pv_value("COMP:D:SP", expected_concentrations[3])
-        self.ca.set_pv_value("START:SP", 1)
+        self.ca.set_pv_value("START:SP", 1, sleep_after_set=1.0)
 
```

Comment:
Would it be clearer to use `sleep(1.0)` with a comment that a `sleep` was needed for the emulator to process the data - I feel adding sleeps inside "set pv" is hiding things and making it less clear why they are being done. Also having a default sleep mechanism inside a `set_pv_value` only used during testing not only slows down the tests, but also potentially hides race conditions. Previously the four concentration commands here would have had a 1 second sleep automatically added in tests, but not in a user script, so if there had been a race condition in db logic (which there isn't) the tests would have missed it.

Reply:
Agreed, we could add the `sleep(1.0)` here and work towards removing the sleep from `set_pv_value` eventually but I want to do it yet 'cos it will just break all the tests again

Reply:
yes, so the plan would be to move towards having tests creating a CaChannel with "default_wait_time=0.0" (it would still be 1.0 if not specified) and have any needed delays as explicit sleep commands with a comment as to why